applications/luci-app-cloudflared/htdocs/luci-static/resources/view/cloudflared/config.js

```diff
@@ -0,0 +1,119 @@
+/* This is free software, licensed under the Apache License, Version 2.0
+ *
+ * Copyright (C) 2024 Hilman Maulana <hilman0.0maulana@gmail.com>
+ */
+
+'use strict';
+'require form';
+'require poll';
+'require rpc';
+'require uci';
+'require view';
+
+var callServiceList = rpc.declare({
+	object: 'service',
+	method: 'list',
+	params: ['name'],
+	expect: { '': {} }
+});
+
+function getServiceStatus() {
+	return L.resolveDefault(callServiceList('cloudflared'), {}).then(function (res) {
+		var isRunning = false;
+		try {
+			isRunning = res['cloudflared']['instances']['cloudflared']['running'];
+		} catch (ignored) {}
+		return isRunning;
+	});
+}
+
+function renderStatus(isRunning) {
+	var spanTemp = '<label class="cbi-value-title">Status</label><div class="cbi-value-field"><em><span style="color:%s">%s</span></em></div>';
+	var renderHTML;
+	if (isRunning) {
+		renderHTML = String.format(spanTemp, 'green', _('Running'));
+	} else {
+		renderHTML = String.format(spanTemp, 'red', _('Not Running'));
+	}
+
+	return renderHTML;
+}
+
+return view.extend({
+	load: function () {
+		return Promise.all([
+			uci.load('cloudflared')
+		]);
+	},
+
+	render: function (data) {
+		var m, s, o;
+
+		m = new form.Map('cloudflared', _('Cloudflared Zero Trust Tunnel'),
+			_('Cloudflare Zero Trust Security services help you get maximum security both from outside and within the network.') + '<br />' +
+			_('Create and manage your cloudflared network on the <a %s>Cloudflare Zero Trust</a> website.')
```

Comment:
```diff
-			_('Create and manage your cloudflared network on the <a %s>Cloudflare Zero Trust</a> website.')
+			_('Create and manage your network on the <a %s>Cloudflare Zero Trust</a> website.')

Reply:
maybe instead of the "website" use "dashboard"?:

    _('Create and manage your network on the <a %s>Cloudflare Zero Trust</a> dashboard.')

Reply:
Acceptable